stratum/hal/lib/phal/phal.proto

```diff
@@ -65,6 +65,14 @@ message PhalCardConfig {
   CachePolicyConfig cache_policy = 3;
 }
 
+message PhalOpticalCardConfig {
```

Comment:
I think we can reuse phal card config.
We also don't need to config frequency, power, ... in th phal config

Reply:
After discussion we decided that having separate messages is helpful to differentiate between SFP cards and optical cards.